src/widget/form/settings/generalform.cpp

```diff
@@ -104,7 +106,9 @@ static QStringList langs = {"Arabic",
                             "Türkçe",
                             "ئۇيغۇرچە",
                             "Українська",
-                            "中文（中国）"};
+                            "中文（中国）",
+                            "Chinese (Taiwan)"
```

Comment:
@LNDDYL I don't know what should be here. Or whether it's in the right position on the list. Help?